Doc/library/shutil.rst

```diff
@@ -263,8 +263,15 @@ Directory and files operations
 
    If *copy_function* is given, it must be a callable that will be used to copy
    each file. It will be called with the source path and the destination path
-   as arguments. By default, :func:`~shutil.copy2` is used, but any function
-   that supports the same signature (like :func:`~shutil.copy`) can be used.
+   as arguments. By default, :func:`~shutil.copy2` is used, but any function
+   that supports the same signature (like :func:`~shutil.copy`) can be used.
+
+   If *dirs_exist_ok* is false (the default) and *dst* already exists, a
+   :exc:`FileExistsError` is raised. If *dirs_exist_ok* is true, the copying
+   operation will continue if it encounters existing directories, and files
+   within the *dst* tree will be overwritten by corresponding files from the
+   *src* tree. Internally, this parameter is passed directly to
+   :func:`os.makedirs`.
```

Comment:
We should avoid making promises about the implementation. Maybe just remove this sentence.

Reply:
Ok, I removed that sentence in c169b1d3d2d67e9be37a95b3843896b0bd3070d9